eslint-local-rules.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
```

Comment:
Isn't this something we'd need in all projects? Shouldn't it live in the sinon eslint project?

Reply:
It is. In the interest of getting this moving, I opted for a bit of duplication, until we figure out how to get it distributed properly.